osx/data/script

```diff
@@ -14,7 +14,10 @@ source activate
 # setup the environment to not mess with the system
 export DYLD_FALLBACK_LIBRARY_PATH="${SCRIPT_PATH}/../lib:$DYLD_FALLBACK_LIBRARY_PATH"
 export LD_PRELOAD_PATH="${SCRIPT_PATH}/../lib"
-export KIVY_HOME="${SCRIPT_PATH}/.kivy"
+BUNDLE_ID=$(ls -1 $(pwd)/../../../MacOS)
```

Comment:
How about using something like `BUNDLE_ID=$(defaults read "${SCRIPT_PATH}/../Info" CFBundleIdentifier)`, which resolves to the app bundle identifier? (in our case is `org.kivy.osxlauncher`)

Reply:
Better idea 👍